Velaptor/InternalExtensionMethods.cs

```diff
@@ -185,7 +185,7 @@ public static void SuppressDisposableTransientWarning<T>(this Container containe
         ///     for the lifetime of the container. It can't be used for selecting a type based
         ///     on runtime conditions.
         /// </param>
-        /// <param name="suppressDisposal"><see langword="true"/> to ignore dispose warnings if the original code invokes dispose.</param>
+        /// <param name="suppressDisposal"><c>true</c> to ignore dispose warnings if the original code invokes dispose.</param>
         /// <remarks>
         ///     This method uses the container's LifestyleSelectionBehavior to select the exact
```

Comment:
Should the underlined text be spelled without spaces? 

```suggestion
        ///     This method uses the container's LifestyleSelectionBehavior to select the exact
```

Reply:
What underlined text?  I don't see anything underlined?

Reply:
This is documentation pulled from the **SimpleInjector** library.  This is fine as is.